src/yang/ietf-twamp.yang

```diff
@@ -848,7 +838,12 @@ module ietf-twamp {
                information in repeat-interval parameter, and the
                parent TWAMP-Control connection for this test
                session is restarted to negotiate a new instance
-               of this TWAMP-Test session.";
+               of this TWAMP-Test session.
+
+               A value of 4,294,967,295 indicates that the test
```

Comment:
Why not use enum value of -1 for enum forever as Juergen suggested? What you did works but is less user friendly.

Reply:
I know that Juergen suggested it, but here is the problem. repeat is defined as unsigned int, with a range of 0..4294967295. An enum is a signed int, with a range of -2147483648 to 2147483647. First of all, we would be mixing a uint with a signed int as part of the union, if we wanted to define a -1. Secondly, how would the server know whether repeat is a uint value or a signed int value?